WorkflowUI/Tests/WorkflowUIObservationTestCase.swift

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2023 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Combine
+import Workflow
+@_spi(WorkflowUIGlobalObservation) import WorkflowUI
+import XCTest
+
+open class WorkflowUIObservationTestCase: XCTestCase {
```

Comment:
there's some scratch work in here, but some ideas for how one could tie into this system and forward events through Combine